### PR TITLE
[dev] Update dev image

### PR DESCRIPTION
**Pull request by andrew-farries**
## Description

Update the dev image to use the latest version of the `workspace-full` image.

Also runs the `kots` installation with `sudo` now that the `kots` installation script has changed to require this.

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`

### CODE REVIEW

File: dev/image/Dockerfile [hunk 1]
```diff
@@ -242,11 +247,6 @@ RUN curl -fsSLO https://github.com/brancz/gojsontoyaml/releases/download/v${GOJS
     tar -tzf gojsontoyaml_${GOJSONTOYAML_VERSION}_linux_amd64.tar.gz | xargs rm -f && \
     rm -f gojsontoyaml_${GOJSONTOYAML_VERSION}_linux_amd64.tar.gz
 
-# Install Replicated and KOTS
-RUN curl https://raw.githubusercontent.com/replicatedhq/replicated/$(curl -s https://api.github.com/repos/replicatedhq/replicated/releases/latest | jq -r '.name')/install.sh | sudo bash && \
-    curl https://kots.io/install | bash && \
```
Comment (mustard-mh) [inline dev/image/Dockerfile:247]: I think add `sudo` before bash is enough

File: dev/image/Dockerfile [hunk 2]
```diff
@@ -128,6 +128,11 @@ RUN install-packages \
 # Install netcat to use it as proxy for SSH access to Harvester VMs
 RUN install-packages netcat
 
+# Install Replicated and KOTS
+RUN curl https://raw.githubusercontent.com/replicatedhq/replicated/$(curl -s https://api.github.com/repos/replicatedhq/replicated/releases/latest | jq -r '.name')/install.sh | sudo bash && \
+    curl https://kots.io/install | bash && \
+    bash -c "echo . \<\(kubectl kots completion bash\) >> ~/.bashrc"
```
Comment (mustard-mh) [inline dev/image/Dockerfile:134]: this will be write in `/root/.bashrc` which is not expected

**Review by mustard-mh — CHANGES_REQUESTED** [review]

**Review by vulkoingim — APPROVED** [review]

**Comment by andrew-farries** [comment]
#13898 was changed to make no modifications to the Dockerfile. This PR is good to go 🚀 .

File: dev/image/Dockerfile (hunk 1 above)
Comment (andrew-farries) [reply dev/image/Dockerfile:247]: Thanks, I've changed to use `sudo` as you suggested.

**Review by nandajavarma — APPROVED** [review]

**Review by Furisto — APPROVED** [review]

**Review by mustard-mh — APPROVED** [review]
LGTM

**Pull request merged**